packages/libs/error-stack/src/frame/frame_impl.rs

```diff
@@ -97,6 +99,96 @@ unsafe impl<A: 'static + Debug + Display + Send + Sync> FrameImpl for PrintableA
     }
 }
 
+#[repr(transparent)]
+#[cfg(feature = "anyhow")]
+struct AnyhowContext(anyhow::Error);
+
+#[cfg(feature = "anyhow")]
+impl fmt::Debug for AnyhowContext {
+    fn fmt(&self, fmt: &mut fmt::Formatter<'_>) -> fmt::Result {
+        fmt::Debug::fmt(&self.0, fmt)
+    }
+}
+
+#[cfg(feature = "anyhow")]
+impl fmt::Display for AnyhowContext {
+    fn fmt(&self, fmt: &mut fmt::Formatter<'_>) -> fmt::Result {
+        fmt::Display::fmt(&self.0, fmt)
+    }
+}
+
+#[cfg(feature = "anyhow")]
+impl Context for AnyhowContext {
+    #[cfg(all(nightly, feature = "std"))]
```

Comment:
why is this `nightly` + `std`?

Reply:
Good catch. Previously this was called on `Error`, which is only implemented on `anyhow` with `std`. I changed it in 7b14474ba149b20a02ef8d5d6280ea9b72ea97e8.
This shouldn't have any effect right now because they only provide `Backtrace`, which requires `std`, but in the case they add more provided values, this is the better option. thank you!

Reply:
I had to revert it as `Provider` is only implemented for `anyhow::Error` on `std`. I added a comment to state this.